bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/VoiceManager.java

```diff
@@ -469,6 +470,25 @@ public TTSService getTTS() {
     }
 
     /**
+     * Retrieves a TTS service with the given id.
+     *
+     * @param id the id of the TTS service
+     * @return a TTS service or null, if no service with this id exists
+     */
+    public TTSService getTTS(String id) {
+        return ttsServices.get(id);
+    }
+
+    /**
+     * Retrieves all TTS services.
+     *
+     * @return a collection of TTS services
+     */
+    public Collection<TTSService> getTTSs() {
+        return Collections.unmodifiableCollection(ttsServices.values());
```

Comment:
You are so right. Shame on me that I do such beginner mistakes...
Should be fixed now.